src/tools/singlejar/BUILD

```diff
@@ -415,7 +416,8 @@ cc_library(
     hdrs = ["test_util.h"],
     deps = [
         "//src/main/cpp/util",
-        "@com_google_googletest//:gtest_main",
+        "@bazel_tools//tools/cpp/runfiles",
```

Comment:
Please, for now, depend on `//tools/cpp/runfiles`.
I'm about to send a patch to the runfiles library that'll fix the issue with the Google-internal use case.
Please also add a TODO with my name (e.g. `TODO(laszlocsomor)`) saying I should use this library from `@bazel_tools`.